### DISCUSSION TITLE
Persist Timer Color Even if the page is refreshed

### DISCUSSION THREAD

**Discussion by SadiPro07** [vydimitrov/react-countdown-circle-timer#245] [Q&A]
in my scenario iam I was using different timers color based on time remaining to complete the task.

so if the timer color switches to red because some task is about to start but when i refresh the page it resets to green color.

is there a way to persist timer color @vydimitrov?

**Comment by vydimitrov** [comment]
Do you change the remaining time after refreshing the page? You should persist the remaining time and after refresh start from it, this way you will get the correct color.
